packages/markdown-slate/src/slateToCiceroMarkDom.js

```diff
@@ -17,6 +17,33 @@
 const NS = 'org.accordproject.commonmark';
 const NS_CICERO = 'org.accordproject.ciceromark';
 
+const lastNode = (input) => input.nodes[input.nodes.length - 1];
+
+const isSecondLastClause = (input) => input.nodes[input.nodes.length - 2]
```

Comment:
This call will still raise an `undefined` error if there are less than two nodes in the input.
I think those would be best written as `input.nodes[input.nodes.length - 1] && input.nodes[input.nodes.length - 1].$class` etc.